desktop/linux/install.md

```diff
@@ -138,7 +127,84 @@ Log out and log back in so that your group membership is re-evaluated.
+there may be situations where running both at the same time can cause issues.
+For example, when mapping network ports (`-p` / `--publish`) for containers, both
+Docker Desktop and Docker Engine may attempt to reserve the same port on your
+machine, which can lead to conflicts ("port already in use").
+
+We generally recommend stopping the Docker Engine while you're using Docker Desktop
+to prevent the Docker Engine from consuming resources and to prevent conflicts
+as described above.
+
+Use the following command to stop the Docker Engine service:
+
+```console
+$ sudo systemctl stop docker docker.socket containerd
+```
+Depending on your installation, the Docker Engine may be configured to automatically
+start as a system service when your machine starts. Use the following command to
+disable the Docker Engine service, and to prevent it from starting automatically:
+
+```console
+$ sudo systemctl disable docker docker.socket containerd
+```
+### Switch between Docker Desktop and Docker Engine
+{: id="context" }
+
+The Docker CLI can be used to interact with multiple Docker Engines. For example,
+you can use the same Docker CLI to control a local Docker Engine and to control
+a remote Docker Engine instance running in the cloud. [Docker Contexts](../../engine/context/working-with-contexts.md)
+allow you to switch between Docker Engines instances.
+
+When installing Docker Desktop, a dedicated "desktop-linux" context is created to
+interact with Docker Desktop. On startup, Docker Desktop automatically sets its
+own context (`desktop-linux`) as the current context. This means that subsequent
+Docker CLI commands target Docker Desktop. On shutdown, Docker Desktop resets
+the current context to the `default` context.
+
+Use the `docker context ls` command to view what contexts are available on your
+machine. The current context is indicated with an asterisk (`*`);
+
+```console
+$ docker context ls
+NAME            DESCRIPTION                               DOCKER ENDPOINT                                  ...
+default *       Current DOCKER_HOST based configuration   unix:///var/run/docker.sock                      ...
+desktop-linux                                             unix:///home/<user>/.docker/desktop/docker.sock  ...        
+```
+
+If you have both Docker Desktop and Docker Engine installed on the same machine,
+you can run the `docker context use` command to switch between the Docker Desktop
+and Docker Engine contexts. For example, use the "default" context to interact
+with the Docker Engine;
+
+```console
+$ docker context use default
+default
+Current context is now "default"
+```
+
+And use the `docker-desktop` context to interact with Docker Desktop:
+
+```console
+$ docker context use docker-desktop
+docker-desktop
+Current context is now "docker-desktop"
+```
+
+Refer to the [Docker Context documentation](../../engine/context/working-with-contexts) for more details.
```

Comment:
Link-checker was failing; at a glance, I think it's missing the `.md` extension here;

```suggestion
Refer to the [Docker Context documentation](../../engine/context/working-with-contexts.md) for more details.
```